### PR TITLE
Add possibility to work with secured mongo databases

### PR DESCRIPTION
**Pull request by abinet**
Issue-38

### CODE REVIEW

File: src/lib/mongo.js [hunk 1]
```diff
@@ -18,13 +18,25 @@ var getDb = function(host, done) {
   }
 
   host = host || localhost;
-  var mongoDb = new Db('local', new MongoServer(host, config.mongoPort));
+  var database = config.databse || "admin"
```
Comment (cvallance) [inline src/lib/mongo.js]: Can you put the default in config.js like the other settings have defaults?

Comment (cvallance) [inline src/lib/mongo.js]: Also, why change the default from `local` to `admin`?

**Comment by abinet** [comment]
Database default moved to config.js as requested and set to "local". So for disabled authentication it will work as it used to.
If authentication is enabled mongo creates user "root" in database "admin". Then property MONGODB_DATABASE is needed to change authentication database.

**Comment by cvallance** [comment]
Thanks!

**Pull request merged**